app/services/forest_liana/ability/permission.rb

```diff
@@ -35,9 +35,10 @@ def is_smart_action_authorized?(user, collection, parameters, endpoint, http_met
         user_data = get_user_data(user['id'])
         collections_data = get_collections_permissions_data
         begin
-          action = find_action_from_endpoint(collection.name, endpoint, http_method).name
+          puts collection.name.gsub('::', '__')
+          action = find_action_from_endpoint(collection.name.gsub('::', '__'), endpoint, http_method).name
```

Comment:
You can replace `collection.name.gsub('::', '__') ` by the helper `ForestLiana.name_for(collection)`.